src/extension/app/plugins/unpublish/unpublish.js

```diff
@@ -47,12 +47,6 @@ export function createUnpublishPlugin(appStore) {
         const { location, status } = appStore;
         const hasSrc = status.edit?.status === 200;
 
-        // double check
-        if (hasSrc && !appStore.isAuthenticated()) {
```

Comment:
Should never be able to call `unpublish` if unauthenticated

Reply:
Actually, same here. We will never get here if not authenticated.. So In think in this case (and the one above in delete) we just need to remove `!appStore.isAuthenticated()` from the condition.

Reply:
Looks like if this condition is hit, than we are toasting `This page still has a source document and cannot be unpublished.` and then cancelling the operation. Is this what we want, I thought this should be allowed?

Reply:
As discussed, leaving as is